src/lib/db/useAppwriteClient.tsx

```diff
@@ -0,0 +1,5 @@
+import { Client } from 'appwrite';
+const client = new Client();
+const porjectId = process.env.APPWRITE_PROJECT_ID;
+
+client.setEndpoint('http://localhost/v1').setProject(`${porjectId}`);
```

Comment:
here should be the deployed appwrite link?

Reply:
yes but deployed only locally for now.

Reply:
You can use [this kind of strategy](https://metaphore.vercel.app/stories/javascript/easy-fetching-on-nextjs) to build the string!

Reply:
wow I found this very useful, I'll  implement from now.